packages/repl/src/lib/workers/bundler/index.js

```diff
@@ -42,7 +42,7 @@ self.addEventListener(
 					// unpkg doesn't set the correct MIME type for .cjs files
 					// https://github.com/mjackson/unpkg/issues/355
 					const compiler = await fetch(`${svelte_url}/compiler.cjs`).then((r) => r.text());
-					eval(compiler);
+					(0, eval)(compiler);
```

Comment:
Why does this witchcraft even work? 😅

Reply:
https://esbuild.github.io/content-types/#direct-eval

Reply:
Naice!